File: 231A - Team/main.py
# https://codeforces.com/problemset/submission/231/225193614
# Python3.8.10 - 92 ms - 0 KB

answer = 0
for _ in range(int(input())):
    m = input().split()
    if (len([str(x) for x in m if str(x)=="1"]) > 1):
        answer += 1

print(answer)

File: 231A - Team/main.php
<?php

/*
    https://codeforces.com/problemset/submission/231/225195209
    PHP 8.1.7 - 60 ms - 28 KB
*/

function input($file) {
    return trim(fgets($file));
}
$file = fopen('php://stdin', 'r');
$answer = 0;
$n = input($file);

for($i = 0; $i < $n; $i++) {
    $_str = explode(' ', fgets($file));
    $a = $_str[0];
    $b = $_str[1];
    $c = $_str[2];
    if($a == 1) {
        if (($b == 1) || ($c == 1)) {
            $answer++;
        }
    } else if ($b == 1) {
        if ($c == 1) {
            $answer++;
        }
    }
}

echo($answer);
fclose($file);
?>